.github/workflows/freertos_demos.yml

```diff
@@ -52,17 +52,17 @@ jobs:
         uses: microsoft/setup-msbuild@v1.1
 
       - env:
-          stepName: Build WIN32-MSVC Full Demo
+          stepName: Build WIN32-64-MSVC Full Demo
```

Comment:
Can we avoid renaming the directory and solution file?

Reply:
Sure we can. This change was made to make clear that this can be build for both x86 and x64 arch.
Let me know if you prefer to keep the old name and I'll revert the related changes.

Reply:
Yes, please.

Reply:
Done!